app/templates/gentelella/guest/event/schedule.html

```diff
@@ -4,10 +4,6 @@
 
 {% set scheduler_readonly = true %}
 
-{% block head_css_bootstrap %}
-    <link rel="stylesheet" href="{{ url_for('static', filename='vendor/bootstrap/dist/css/bootstrap-noprint.min.css') }}">
```

Comment:
Can you please check the export functionality whether that works fine without this or not? And create a pretty wide and long schedule to check. So that you need to scroll both horizontally and vertically and then check.